libs/curie.js

```diff
@@ -2,34 +2,52 @@
  * Abstract base for mapping for CURIE-style identifiers (e.g. EFO:0002067) to links.
  */
 export default class Curie {
+  #curie
+  #prefix
+  #id
+
   constructor(curie) {
```

Comment:
I think I’ll leave this as is since any invalid curies would be very rare, as the schemas have strict regexes controlling these. The only realistic error situation would involve a schema with a curie the front end didn’t know about, in which case we’d still be able to display the original string, just without a link.

I need to rebase this, so this will probably show up for code review again.